build-support/bin/ci.py

```diff
@@ -107,7 +109,12 @@ def create_parser() -> argparse.ArgumentParser:
   parser.add_argument("--rust-tests", action="store_true", help="Run Rust tests.")
   parser.add_argument("--jvm-tests", action="store_true", help="Run JVM tests.")
   parser.add_argument(
-    "--integration-tests", action="store_true", help="Run Python integration tests."
+    "--integration-tests-v1", action="store_true",
+    help="Run Python integration tests w/ V1 runner."
+  )
+  parser.add_argument(
+    "--integration-tests-v2", action="store_true",
+    help="Run Python integration tests w/ V2 runner (and possible remoting)."
```

Comment:
Nit: Is this "and possible remoting", or do we try to do "remoting where possible"?

The difference is that if I read the first option, it doesn't help me know what to expect, whereas with the second option I can expect certain things to be ran under remoting, and be surprised if they aren't.